scripts/apps/authoring-react/toolbar/multi-edit-toolbar-action.tsx

```diff
@@ -27,6 +27,10 @@ export class MultiEditToolbarAction extends React.Component<IProps, IState> {
     }
 
     render(): ReactNode {
+        const options = sdApi.article.getWorkQueueItems().filter((article) =>
```

Comment:
Either inline it or move inside `getOptions` function so it only executes when `getOptions` is called.